src/themes/playful.css

```
@@ -334,14 +350,415 @@
+      --button-shadow-inner-a: var(--wa-color-green-30);
+      --button-shadow-inner-b: var(--wa-color-green-80);
+      --button-text-shadow: var(--wa-color-green-40);
+    }
+
+    &[variant='warning'] {
+      --button-gradient-top: var(--wa-color-yellow-40);
+      --button-gradient-middle: var(--wa-color-yellow-50);
+      --button-gradient-bottom: var(--wa-color-yellow-60);
+      --button-shadow-outer: var(--wa-color-yellow-60);
+      --button-shadow-inner-a: var(--wa-color-yellow-30);
+      --button-shadow-inner-b: var(--wa-color-yellow-80);
+      --button-text-shadow: var(--wa-color-yellow-40);
+    }
+
+    &[variant='danger'] {
+      --button-gradient-top: var(--wa-color-red-40);
+      --button-gradient-middle: var(--wa-color-red-50);
+      --button-gradient-bottom: var(--wa-color-red-60);
+      --button-shadow-outer: var(--wa-color-red-60);
+      --button-shadow-inner-a: var(--wa-color-red-30);
+      --button-shadow-inner-b: var(--wa-color-red-80);
+      --button-text-shadow: var(--wa-color-red-40);
+    }
+
+    &[variant='neutral'] {
+      --button-gradient-top: var(--wa-color-base-40);
+      --button-gradient-middle: var(--wa-color-base-50);
+      --button-gradient-bottom: var(--wa-color-base-60);
+      --button-shadow-outer: var(--wa-color-base-60);
+      --button-shadow-inner-a: var(--wa-color-base-30);
+      --button-shadow-inner-b: var(--wa-color-base-80);
+      --button-text-shadow: var(--wa-color-base-40);
+    }
+
+    &::part(base) {
+      transition: all 0.05s ease-out;
+    }
+
+    &::part(base):hover {
+      transform: scale(1.02, 1.02);
+      background: var(--background-hover);
+    }
+
+    &::part(base):active {
+      background: var(--background-active);
+      --box-shadow: inset 0 -2px 8px var(--button-shadow-inner-a), inset 0 2px 8px var(--button-shadow-inner-b);
+      transform: scale(0.98, 0.98);
+    }
+  }
+
+  wa-button::part(label),
+  wa-button wa-icon {
+    font-weight: var(--wa-font-weight-black);
+    text-shadow: 0 2px 2px var(--button-text-shadow);
+  }
+
+  wa-button[outline] {
+    --box-shadow: none;
+    --border-width: var(--wa-form-controls-border-width);
+    --label-color: var(--wa-color-neutral-text-on-fill);
+    --label-color-hover: var(--wa-color-neutral-text-on-spot);
+
+    &::part(label) {
+      text-shadow: none;
+    }
+  }
+
+  wa-card {
+    --border-radius: var(--wa-panel-corners);
+  }
+
+  wa-card img {
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+
+  wa-carousel img {
+    border-radius: var(--wa-panel-corners);
+    box-shadow: none;
+  }
+
+  wa-alert {
+    --border-radius: var(--wa-corners-m);
+    --border-style: dashed;
+    --border-width: var(--wa-form-controls-border-width);
+    --icon-size: var(--wa-font-size-m);
+    --padding: var(--wa-space-s);
+    font-size: var(--wa-font-size-s);
+
+    &::part(message) {
+      padding-left: var(--wa-space-l);
+    }
+  }
+
+  wa-tag {
```

```suggestion
  wa-tag {
      --border-radius: var(--wa-corners-pill);
```
We'll probably want to add this. However, it looks like `wa-tag` isn't currently respecting its `--border-radius` property as it should. I can push a commit to fix that.